conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,298 @@
+                                   # Ports: 21001 - (21001+Number of CVGs)
+      - tcp://data2-node1:21001    # Same thing for the second pod
+      - tcp://data1-node2:21001    # of for Pods on the other nodes
+ 
+    confd:
+      endpoints:
+      - tcp://data1-node1:22002    # Format: <protocol>://<hostname_ip>:22002
+      - tcp://data1-node2:22002       
+      - tcp://data1-node3:22002    
+      
+    limits:                        # NOTE: This section is different from solution config
+      num_services: 1
+    - name: ios
+      mem: 1Gi-2Gi
+      cpu: <Min>-<Max>
+        
+    clients:                       # Motr Client Settings     
+    num_clients: 2                 # Number of Clients
+    - name: rgw_s3                 # rgw_s3|s3|cclient (Motr Clients)
+      num_instances: 1             # Client instances per node (pod)
+      endpoints:
+      - tcp://server1-node1:21001  # Format: <protocol>://<hostname_ip>:21001 
+      - tcp://server1-node2:21001  # Num endpoints will be same as num of client instances
+                                   # Port range would be 21001 - 21001+num_instances
+    - name: rgw_admin             
+      num_instances: 1           
+      endpoints:
+      - tcp://server1-node1:21001  
+      - tcp://server1-node2:21001  
+      
+    - name: cclient             
+      num_instances: 0             # Optional client. Only for Dev.         
+      endpoints:
+      - tcp://server1-node1:21001  
+      - tcp://server1-node2:21001    
+
+  utils:
+    message_bus_backend: kafka
+
+cluster:
+  id: 0007ec45379e36d9fa089a3d615c32a3
+  name: cortx-cluster
+  storage_set_count: 1
+  storage_set:
+  - name: storage-set-1
+    durability:
+      dix:
+        data: '1'
+        parity: '0'
+        spare: '0'
+      sns:
+        data: '1'
+        parity: '0'
+        spare: '0'
+    nodes:
+    - 9995f539f4f770e2a3fe9e2e615c32a8
+    - 1115f539f4f770e2a3fe9e2e615c32a8
+    - aaa120a9e051d103c164f605615c32a4
+    - bbb340f79047df9bb52fa460615c32a5
+    - ccc8700fe6797ed532e311b0615c32a7
+    - ddd120a9e051d103c164f605615c32a4
+    - eee340f79047df9bb52fa460615c32a5
+    - fff8700fe6797ed532e311b0615c32a7  
+
+node:
+  1115f539f4f770e2a3fe9e2e615c32a8:
+    type: ha_node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    components:
+    num_components: 2
+    - name: utils
+      version: 2.0.0-33
+    - name: ha
+      version: 2.0.0-47
+    hostname: ha-node
+    name: ha-node
+
+    provisioning:
+      phase: deployment
+      status: success
+      version: 2.0.0-576
+    storage_set: storage-set-1
+
+  9995f539f4f770e2a3fe9e2e615c32a8:
+    type: control_node
+    hostname: control-node
+    name: control-node
+    cluster_id: 0007ec45379e36d9fa089a3d615c32a3
+    num_components: 4
+    components:
+    - name: utils
+      version: 2.0.0-33
+    - name: motr
+      services:
+      - fsm
+      version: 2.0.0-33
+    - name: s3
```

Comment:
line 209-211 need to be cleaned up, as bgscheduler service related to legacy s3server is no more needed

Reply:
done

Reply:
done